UnicornRender/video/source/vulkan/Renderer.cpp

```diff
@@ -403,11 +368,41 @@ void Renderer::OnMeshMaterialUpdated(Mesh* mesh, VkMesh* vkMesh)
     CreateCommandBuffers();
 }
 
+void Renderer::AddMesh(Mesh* mesh)
+{
+    assert(nullptr != mesh);
+
+    auto vkmesh = new VkMesh(m_vkLogicalDevice, m_vkPhysicalDevice, m_commandPool, m_graphicsQueue, *mesh);
+    if (!AllocateMaterial(*mesh, *vkmesh))
+    {
+        LOG_ERROR("Can't allocate material!");
```

Comment:
What is the point to continue execution of the function if the memory allocation failed?